packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -637,14 +638,19 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
 
         uint256 totalVotes = voteInstance.results.totalVotes;
         uint256 totalCorrectVotes = voteInstance.results.getTotalCorrectlyVotedTokens();
-        uint256 stakedAtRound = rounds[lastVotingRound].cumulativeStakeAtRound;
```

Comment:
why this variable needs to be changed?

Reply:
it seemed more correctly named like this given the other variables of `totalVotes` `totalCorrectVotes` then the correct variable should be `totalStaked` not `stakedAtRound`, given the context.